waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -163,19 +163,27 @@ genGitignore =
 genSrcDir :: AppSpec -> Generator [FileDraft]
 genSrcDir spec =
   sequence
-    [ return $ C.mkSrcTmplFd $ C.asTmplSrcFile [relfile|app.js|],
-      return $ C.mkSrcTmplFd $ C.asTmplSrcFile [relfile|utils.js|],
-      return $ C.mkSrcTmplFd $ C.asTmplSrcFile [relfile|core/AuthError.js|],
-      return $ C.mkSrcTmplFd $ C.asTmplSrcFile [relfile|core/HttpError.js|],
-      genDbClient spec,
+    [ genDbClient spec,
       genConfigFile spec,
       genServerJs spec
     ]
+    <++> genDirectCopies
     <++> genRoutesDir spec
     <++> genOperationsRoutes spec
     <++> genOperations spec
     <++> genAuth spec
 
+genDirectCopies :: Generator [FileDraft]
+genDirectCopies =
```

Comment:
I do understand that `mkSrcTmplFd` is confusing name, instead we could maybe name it `directCopyFromTemplate` or smth like that, but `genDirectCopies` doesn't seem like a solution to me. It generates direct copies. Of what? To where? Why? It is too much out of context, being a standalone function at the top level like this. If you put it into `where` of `genSrcDir`, I think that would already help a bit. But I would like it even better if it would take a list of paths. Can we think of different ways to try and improve the code here, especially regarding the old naming?

Reply:
We talked about this in person. All changed should be implemented.